ui/app/components/ui/truncated-definition-list/truncated-definition-list.js

```diff
@@ -0,0 +1,74 @@
+import { pick } from 'lodash'
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import { COLORS, SIZES } from '../../../helpers/constants/design-system'
+import Box from '../box'
+import Button from '../button'
+import DefinitionList from '../definition-list/definition-list'
+import Popover from '../popover'
+
+export default function TruncatedDefinitionList({
+  dictionary,
+  tooltips,
+  prefaceKeys,
+  title,
+}) {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false)
+
+  return (
+    <div className="truncated-definition-list">
+      <Box
+        margin={6}
+        padding={4}
+        paddingBottom={3}
+        borderRadius={SIZES.LG}
+        borderColor={COLORS.UI2}
+      >
+        <DefinitionList
+          dictionary={pick(dictionary, prefaceKeys)}
+          tooltips={tooltips}
+        />
+        <Button
+          className="truncated-definition-list__view-more"
+          type="link"
+          onClick={() => setIsPopoverOpen(true)}
+        >
+          View all details
```

Comment:
This should be using a localized message

Reply:
resolved in latest